lib/screens/home_screen/overview/widgets/contact_section.dart

```diff
@@ -23,7 +23,7 @@ import 'package:picos/themes/global_theme.dart';
 
 /// Widget which shows a graph
 class ContactSection extends StatelessWidget {
```

Comment:
Please rename 'ContactSection' or change comment

Reply:
resolved in [refactoring (Add comments)](https://github.com/healthcare-it-solutions/picos/pull/184/commits/26d5c6e60feb8cb2eee7b9477f1a1b5f7db36249)